ci/scripts/deterministic-e2e-test.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+
+# Exits as soon as any line fails.
+set -euo pipefail
+
+source ci/scripts/common.env.sh
+
+echo "--- Generate RiseDev CI config"
+cp ci/risedev-components.ci.env risedev-components.user.env
+
+echo "--- Build deterministic simulation e2e test runner"
+cargo make sslt --release -- --help
```

Comment:
add timeout for each step so that Buildkite will terminate the task when timed out instead of retrying.